### DISCUSSION TITLE
Is the 'creatable' option permanently removed from Select and Multiselect components?

### DISCUSSION THREAD

**Discussion by Sajarin-M** [mantinedev/mantine#4982] [Q&A] [answered]
Has the "creatable" option been permanently removed from the Select and Multiselect components, or is there a plan to reintroduce it in future minor releases? 

Should we consider creating a custom component using the base Combobox component if we desire this feature in version 7?

It was a nice feature.

**Comment by rtivital** [comment, marked as answer]
Yes, the creatable prop was permanently removed, you should build it on your side with Combobox component

**Reply by AFusco** [reply]
This should be written in the changelog

**Comment by jacobwilson41** [comment]
you can also create a createable option using the `nothingFoundMessage` prop

```
<MultiSelect
      nothingFoundMessage={<Combobox.Option value="$create">+ Create {searchValue}</Combobox.Option>}
/>
```
